code/modules/paperwork/photography.dm

```diff
@@ -149,24 +150,15 @@
 /obj/item/device/camera/attack(mob/living/carbon/human/M, mob/user)
 	return
 
-/obj/item/device/camera/attack_self(mob/user)
-	..()
-	on = !on
-	if(on)
-		src.icon_state = icon_on
-	else
-		src.icon_state = icon_off
-	to_chat(user, "You switch the camera [on ? "on" : "off"].")
```

Comment:
One problem I noticed with cameras not being disable-able is that you can't put them down onto a table/rack/etc. without taking a picture of it.

(I can't actually think of a useful fix at the moment (late night sleepy brain), but I thought I'd mention this now anyway.)

Reply:
Not the biggest issue now that camera film can be easily restored via vendor (plus its already everywhere shipside (and I even think req has some)). I'd rather people be taking more pictures than less pictures always since missing out on funny photos can feel bad

Reply:
The switching on/off functionality was to address this. An alternative would be to require both hands to take a photo.

Reply:
> The switching on/off functionality was to address this. An alternative would be to require both hands (wielding it) to take a photo.

This would be very cute, I'll get spriting for that now